infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java

```diff
@@ -72,15 +85,27 @@ public KubernetesNamespaceFactory(
       @Nullable @Named("che.infra.kubernetes.namespace.default") String defaultNamespaceName,
```

Comment:
It's a good time to mention in docs that `che.infra.kubernetes.namespace` and `che.infra.openshift.project` MUST NOT be updated anymore.

And `che.infra.kubernetes.namespace.default` must be propagated to deployments - like helm chart has `chenamespaces` parameter (I'm OK with handling this as a separate issue, we just should track it).

And maybe the corresponding attribute should be propagated to Che-operator.
Currently, target namespace depends on OpenShift Oauth https://github.com/eclipse/che-operator/blob/master/pkg/deploy/che_configmap.go#L89
With these changes it becomes always `<username>-che`. It's pretty OK with OpenShift oAuth but not sure that it even possible with Che Server service account.

Reply:
I see that it should be resolved now